apps/system/js/app_usage_metrics.js

```diff
@@ -544,6 +546,15 @@
               new Date(newStartTime).toString());
       }
       break;
+
+      case IACMETRICS:
```

Comment:
nit: it looks like the indentation is off on the case block, I'm not sure if github is just playing tricks with the code though?